src/content/learn/editor-setup.md

```diff
@@ -1,62 +1,62 @@
 ---
-title: Editor Setup
+title: એડિટર સેટઅપ
 ---
 
 <Intro>
 
-A properly configured editor can make code clearer to read and faster to write. It can even help you catch bugs as you write them! If this is your first time setting up an editor or you're looking to tune up your current editor, we have a few recommendations.
+યોગ્ય રીતે ગોઠવેલ એડિટર કોડને વાંચવા માટે સ્પષ્ટ અને લખવા માટે ઝડપી બનાવી શકે છે. તે તમને ભૂલો લખતા જ તેને પકડવામાં પણ મદદ કરી શકે છે! જો આ તમે પ્રથમ વખત એડિટર સેટઅપ કરી રહ્યાં છો અથવા તમે તમારા વર્તમાન એડિટર ને ટ્યુન કરવા માટે શોધી રહ્યાં છો, તો અમારી પાસે કેટલીક ભલામણો છે.
 
 </Intro>
 
 <YouWillLearn>
 
-* What the most popular editors are
-* How to format your code automatically
+* સૌથી વધુ લોકપ્રિય એડિટરો શું છે
+* તમારો કોડ આપમેળે કેવી રીતે ફોર્મેટ કરવો
 
 </YouWillLearn>
 
-## Your editor {/*your-editor*/}
+## તમારુ એડિટર {/*your-editor*/}
 
-[VS Code](https://code.visualstudio.com/) is one of the most popular editors in use today. It has a large marketplace of extensions and integrates well with popular services like GitHub. Most of the features listed below can be added to VS Code as extensions as well, making it highly configurable!
+[VS Code](https://code.visualstudio.com/) એ આજે ઉપયોગમાં લેવાતા સૌથી લોકપ્રિય એડિટરો પૈકીનું એક છે. તેની પાસે એક્સ્ટેંશનનું વિશાળ બજાર છે અને GitHub જેવી લોકપ્રિય સેવાઓ સાથે સારી રીતે સંકલિત છે. નીચે સૂચિબદ્ધ મોટાભાગની સુવિધાઓ VS Code માં એક્સ્ટેંશન તરીકે પણ ઉમેરી શકાય છે, જે તેને અત્યંત રૂપરેખાંકિત બનાવે છે!
 
-Other popular text editors used in the React community include:
+React સમુદાયમાં વપરાતા અન્ય લોકપ્રિય ટેક્સ્ટ એડિટરો માં નીચેનાનો એડિટરો નો સમાવેશ થાય છે:
 
-* [WebStorm](https://www.jetbrains.com/webstorm/) is an integrated development environment designed specifically for JavaScript.
-* [Sublime Text](https://www.sublimetext.com/) has support for JSX and TypeScript, [syntax highlighting](https://stackoverflow.com/a/70960574/458193) and autocomplete built in.
-* [Vim](https://www.vim.org/) is a highly configurable text editor built to make creating and changing any kind of text very efficient. It is included as "vi" with most UNIX systems and with Apple OS X.
+* [WebStorm](https://www.jetbrains.com/webstorm/) એક સંકલિત વિકાસ પર્યાવરણ છે જે ખાસ કરીને JavaScript માટે રચાયેલ છે.
+* [Sublime Text](https://www.sublimetext.com/) JSX અને TypeScript સપોર્ટ કરે છે,[સિન્ટેક્સ હાઇલાઇટિંગ](https://stackoverflow.com/a/70960574/458193) અને સ્વતઃપૂર્ણ બિલ્ટ ઇન માટે સપોર્ટ ધરાવે છે.
+* [Vim](https://www.vim.org/) એ અત્યંત રૂપરેખાંકિત ટેક્સ્ટ એડિટર છે જે કોઈપણ પ્રકારના ટેક્સ્ટને ખૂબ જ કાર્યક્ષમ બનાવવા અને બદલવા માટે બનાવવામાં આવ્યું છે. તે મોટાભાગની UNIX સિસ્ટમો અને Apple OS X સાથે vi તરીકે સમાવિષ્ટ છે.
 
-## Recommended text editor features {/*recommended-text-editor-features*/}
+## ભલામણ કરેલ ટેક્સ્ટ એડિટર સુવિધાઓ {/*recommended-text-editor-features*/}
```

Comment:
I would just not to translate passive voice sentences directly in Gujarati.
Change it to this "ટેક્સ્ટ એડિટર ની સુવિધાવો માટેની ભલામણ"